glide.lock

```diff
@@ -1876,7 +1876,7 @@ imports:
   - pkg/util/proto/testing
   - pkg/util/proto/validation
 - name: k8s.io/kubernetes
-  version: 91becade863adc3e9a01585d027896fd1f0d12a7
+  version: c388e666caa7903dff9d4da6a4f850346d402da3
```

Comment:
@deads2k remind me again why this shouldn't worry me?

Reply:
it's because we sync levels back.  As long as you don't see content change in k8s.io/kube, this is ok.